### PR TITLE
Backport of 2018 PbPb vertex smearing parameters

### PR DESCRIPTION
**Pull request by ColeLindsey**
Backport of https://github.com/cms-sw/cmssw/pull/25390

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

File: Configuration/StandardSequences/python/VtxSmeared.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
     'Realistic100ns13TeVCollisionBetaStar90mLowBunches' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic100ns13TeVCollisionBetaStar90mLowBunches_cfi',
     'Realistic25ns13TeVEarly2017Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2017Collision_cfi',
     'Realistic25ns13TeVEarly2018Collision' : 'IOMC.EventVertexGenerators.VtxSmearedRealistic25ns13TeVEarly2018Collision_cfi',
-
+    'RealisticPbPbCollision2018' : 'IOMC.EventVertexGenerators.VtxSmearedRealisticPbPbCollision2018_cfi',
 }
 VtxSmearedDefaultKey='Realistic50ns13TeVCollision'
-VtxSmearedHIDefaultKey='RealisticHICollision2015'
+VtxSmearedHIDefaultKey='RealisticPbPbCollision2018'
```
Comment (fabiocos) [inline Configuration/StandardSequences/python/VtxSmeared.py]: @ColeLindsey @civanch this was not updated in master, why? I think we would like to have 104X and 103X consistent, right?

Comment (ColeLindsey) [reply Configuration/StandardSequences/python/VtxSmeared.py]: @fabiocos Yes, you're right. Updating this default value was mentioned in an email that I didn't see until after I'd done the last update for the PR for master. The same email mentioned that it was rarely used so I didn't want to risk delaying that PR any longer. I did it on this backport and then figured I could do it on the master later. I hope that's okay.

**Comment by fabiocos** [comment]
@ColeLindsey could you please open a PR that fixes it in master? Thanks

**Comment by ColeLindsey** [comment]
@fabiocos 
Just made a PR to fix it! https://github.com/cms-sw/cmssw/pull/25456

Thank you

**Comment by srimanob** [comment]
@fabiocos @ColeLindsey 
Can we expect this PR to be merged for the next 10_3? This will be need for PbPb GS.

Thanks,

**Comment by fabiocos** [comment]
+operations

**Comment by fabiocos** [comment]
+1

master is being aligned with this PR

**Pull request merged**